components/bl-pdfViewer-component/src/subcomponents.js

```diff
@@ -32,28 +58,31 @@ export function Controls(props) {
       <button
         className="controls-button"
         onClick={ () => setPageIndex(state => state - 1) }
-        disabled={ (pageIndex > numPages) || (pageIndex <= 1) }>
+        disabled={ (pageIndex > pageCount) || (pageIndex <= 1) }>
         <PrevButtonIcon/>
       </button>
       <div className="pages-info">
         <input
           ref={ inputRef }
           className="page-input"
-          value={ pageIndex }
-          onChange={ handlerPageChange }
+          defaultValue={ 1 }
```

Comment:
Looks like we doesn't make it better, and it's still controlled but via the ref. Let revert to using state and controlled input, it's more obvious. Sorry for inconveniences.